NewSpring/corinth
# fix: Updates Series Children Properly

## DESCRIPTION

### What does this PR do, or why is it needed?

This will retrieve the correct content children for studies and sermon series items.

### How do I test this PR?

Boot up the app and pull up a series. Delete one of the children, should disappear in the app on the next pull.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
